src/userscores/qml/MuseScore/UserScores/internal/ExportOptionsView.qml

```diff
@@ -0,0 +1,270 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import MuseScore.UserScores 1.0
+import MuseScore.UiComponents 1.0
+
+ColumnLayout {
+    id: root
+    required property ExportDialogModel exportModel
+
+    spacing: 12
+    property var firstColumnWidth: 72
+
+    ExportOptionItem {
+        firstColumnWidth: root.firstColumnWidth
+        text: qsTrc("userscores", "Format:")
+
+        StyledComboBox {
+            id: fileTypeComboBox
+            Layout.fillWidth: true
+
+            model: [
+                { textRole: qsTrc("userscores", "PDF File"), valueRole: "pdf" },
+                { textRole: qsTrc("userscores", "PNG Images"), valueRole: "png" },
+                { textRole: qsTrc("userscores", "SVG Images"), valueRole: "svg" },
+                { textRole: qsTrc("userscores", "MP3 Audio"), valueRole: "mp3" },
+                { textRole: qsTrc("userscores", "WAV Audio"), valueRole: "wav" },
+                { textRole: qsTrc("userscores", "OGG Audio"), valueRole: "ogg" },
+                { textRole: qsTrc("userscores", "FLAC Audio"), valueRole: "flac" },
+                { textRole: qsTrc("userscores", "MIDI File"), valueRole: "mid" },
+                { textRole: qsTrc("userscores", "MusicXML"), valueRole: "xml" }
+            ]
+
+            maxVisibleItemCount: count
+
+            textRoleName: "textRole"
+            valueRoleName: "valueRole"
+
+            currentIndex: {
+                if (exportModel.selectedExportSuffix === "musicxml" || exportModel.selectedExportSuffix === "mxl") {
+                    return indexOfValue("xml")
+                } else {
+                    return indexOfValue(exportModel.selectedExportSuffix)
+                }
+            }
+
+            onValueChanged: {
+                if (value === "xml") {
+                    exportModel.selectedExportSuffix = "mxl";
+                } else {
+                    exportModel.selectedExportSuffix = value
+                }
+            }
+        }
+    }
+
+    ExportOptionItem {
+        visible: exportModel.selectedExportSuffix === "pdf"
```

Comment:
I think we need the enum on the model for formats

Reply:
Also better to add a loader instead of creating all pages for formats and setting visible by changing the format

Reply:
And move all pages to separate files